stdlib/tarfile.pyi

```diff
@@ -129,7 +131,7 @@ class TarFile:
     def __iter__(self) -> Iterator[TarInfo]: ...
     @classmethod
     def open(
-        cls,
+        cls: Type[_TF],
```

Comment:
Nit: `type[_TF]`

Reply:
mypy doesn't like `type[_TF]` because of https://github.com/python/mypy/issues/10303

Reply:
Sorry about that. I actually tested it before this comment, but it seems that it worked in my small test for some reason, but not here. 😕

Reply:
I have switched back to `Type[_TF]` for now.